home-manager/nvim/which-key.lua

```diff
@@ -18,67 +18,19 @@ require("which-key").setup({
 			g = true, -- bindings for prefixed with g
 		},
 	},
-	-- add operators that will trigger motion and text object completion
```

Comment:
Loads of options changed upstream, but these were all set to their defaults anyway.